clickhouse/migrations/schema_47_cache_to_request_response_rmt.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE request_response_rmt
+ADD COLUMN IF NOT EXISTS cache_reference_id UUID
```

Comment:
We will probably want to add a default here right? Does clickhouse let you do that>